packages/css/README.md

```diff
@@ -0,0 +1,43 @@
+# CSS definitions of the web platform
+
+This package contains CSS property definitions scraped from the latest versions of web platform specifications in [webref](https://github.com/w3c/webref), with fixes applied to ensure ([almost](#guarantees)) all CSS value definitions can be parsed with [CSSTree](https://github.com/csstree/csstree).
```

Comment:
Do we still have this guarantee? If not this and the guarantees section below needs updating.

Reply:
That is why I added the "(almost)" with a link to the guarantees section, which explains that there remains a handful of exceptions in practice.